python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -78,96 +73,14 @@ class MatchOptimisticBConfig(pexConfig.Config):
         dtype=float,
         default=0.02,
     )
-    minSnr = pexConfig.Field(
-        dtype=float,
-        doc="Minimum allowed signal-to-noise ratio for sources used for matching "
-        "(in the flux specified by sourceFluxType); <=0 for no limit",
-        default=40,
+    sourceSelector = sourceSelectorRegistry.makeField(
+        doc="How to select sources for cross-matching",
+        default="matcher"
     )
 
-
```

Comment:
A yep. I noticed a missing double space in front of the main class so I have added that.